packages/cf.js/src/structs.ts

```diff
@@ -0,0 +1,38 @@
+import { ABIEncoding, Address } from "./simple-types";
+
+export enum NodeMessageType {
+  INSTALL = "install",
+  QUERY = "query",
+  ERROR = "error"
+}
+
+export enum QueryType {
+  GET_APP_INSTANCES = "getAppInstances"
+}
+
+export interface MessageDataQuery {
+  queryType: QueryType;
+  appInstances?: any[];
```

Comment:
@mvanderh Why do we use `any` if we have `AppInstance` defined?

Reply:
@mvanderh Is there anything to structure it somehow? Maybe an interface or a type? I tend to think that using `any` on a strongly-typed environment is kinda weird.

Reply:
Good idea.